lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -324,6 +326,36 @@ def post(self, request):
             uname = request.POST.get('student_uname', '').strip()
             self.msg = u'<h4>{0}</h4><p>{1}</p><hr />{2}'.format(
                 _('Delete User Results'), self.delete_user(uname), self.msg)
+        elif action == 'rename_user':
+            old_uname = request.POST.get('old_uname', '').strip()
```

Comment:
I'd at least like to change `uname` to `username`; while we're renaming, could we use a prefix style naming convention, like `username_new` and `username_old`?